docs/content/how-to/aws/index.md

```diff
@@ -0,0 +1,1035 @@
+    style CL fill:#BAE0E9,stroke:#333,stroke-width:3px
+    style SC fill:#BAE0E9,stroke:#333,stroke-width:3px
+
+
+    HO ==> SVC
+    HO ==> RT
+    HO ==> CMHCP
+    HO ==> PVC
+    HO ==> SCS
+    HO ==> APPS
+    HO ==> HCP
+    HO ==> AWSCL
+    HO ==> CL
+    HO ==> SC
+
+    subgraph OMC[Openshift Management Cluster]
+    direction LR
+        subgraph HONS[Hypershift Operator Namespace]
+            HO[fa:fa-server Hypershift Operator]
+        end
+        subgraph CANS[Cluster Namespace]
+            subgraph SC[Secrets]
+                KBCFGSC[admin-kubeconfig]
+                KADPWD[kubeadmin-password]
+            end
+        end
+        subgraph HCPN[Hosted Control Plane Namespace]
+            direction TB
+            subgraph SVC[Services]
+                CIPSVC[Cluster IP Services]
+                LBSVC[LoadBalancer Services]
+            end
+
+            subgraph RT[Routes]
+                ignition-Server
+                ovn-sbdb
+                oauth
+                konnectivity-server
+            end
+
+            subgraph CMHCP[ConfigMaps]
+                Configurations
+                Certificates
+                ...
+            end
+
+            subgraph SCS[Secrets]
+                KBC[Kubeconfig]
+                CA[Certificates]
+                SCETC[...]
+            end
+
+            subgraph PVC[PVCs]
+                etcd
+                ovn
+            end
+
+            subgraph APPS[ControlPlane Apps]
+                STS[Stateful Sets]
+                DP[Deployments]
+                RS[Replica Sets]
+            end
+
+            HCP <-.-> CL <-.-> AWSCL
+            subgraph HCP[Hosted Control Plane]
+                DescHCP("Contains the OCP
+                HostedCluster details")
+            end
+            subgraph AWSCL[AWSCluster]
+                DescAWSC("Contains the AWS details like
+                VPC, Region, Subnet, etc...")
+            end
+            subgraph CL[Cluster]
+                DescCL("Perform the relationship between
+                AWSCluster and HostedControlPlane")
+            end
+        end
+    end
+```
+!!! note "Legend"
+    - **Orange:** OCP objects directly related with AWS.
+    - **Cyan:** Provider agnostic OCP objects.
+    - The *dotted* lines are associations among AWS Components created by the CLI or the Operator
+    - The **thick** lines are objects created by the CLI into AWS platform and needed by the Hypershift Operator to work properly
+
+At the same time, the Hypershift operator will expect to exists the mentioned Infra components in the defined provider (You could use the Hypershift CLI in order to create them automatically). As a first step it will assign an InfraID to all the objects created in the AWS side.
+
+- **Diagram:** The Diagrams shows 2 things, the objects Needed and Created in the AWS infrastructure. After this one we will dissect the objects in a more detailed way.
```

Comment:
I'm confused about the ingress controller in that picture. FWIW The ingress controller runs in the hosted cluster.

Reply:
Totally right. My bad